onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -185,8 +185,8 @@ void upsampleBilinear(
     float in_y = std::min(y / height_scale, static_cast<float>(input_height - 1));
     const int64_t in_y1 = std::min(static_cast<int64_t>(in_y), input_height - 1);
     const int64_t in_y2 = std::min(in_y1 + 1, input_height - 1);
-    dy1[y] = fabs(in_y - in_y1);
-    dy2[y] = fabs(in_y - in_y2);
+    dy1[y] = std::fabs(in_y - in_y1);
```

Comment:
LGTM.

Also, they are not the same function.
In this case, std::fabs indeed is fabsf, not fabs.

Reply:
I believe fabsf is functionally equivalent to fabs in this case as a float will convert to double without loss. In VC++ fabsf just calls fabs with some explicit casts.

        _Check_return_ __inline float __CRTDECL fabsf(_In_ float _X)
        {
            return (float)fabs(_X);
        }